### DISCUSSION TITLE
Why are all my buttons duplicated?

### DISCUSSION THREAD

**Discussion by salawatbro** [filamentphp/filament#10872] [General]
Why are all my buttons duplicated?

<img width="640" alt="image" src="https://github.com/filamentphp/filament/assets/70514229/68c1497e-8119-4f9f-9d6e-2609488bd25b">

<img width="960" alt="image" src="https://github.com/filamentphp/filament/assets/70514229/99a7401c-8a9b-4725-875e-7d145af893cb">

<img width="482" alt="image" src="https://github.com/filamentphp/filament/assets/70514229/fabfe7e2-ce93-444c-a6e9-b5d932932dcb">

**Comment by salawatbro** [comment]
Laravel: 10.41.0
PHP: 8.2.13
Filament: v3.2.2

**Comment by kakajansh** [comment]
In my case it was a cache issue. Everything way okay locally but in production environment buttons were duplicated.

- Make sure you have correct ownership and permissions for project folders
`chown -R www-data:www-data /app`
- [Clear the filament cache](https://filamentphp.com/docs/3.x/panels/installation#improving-filament-panel-performance)
`php artisan filament:optimize-clear`
